cmd/skaffold/app/cmd/config/config.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+type Config = []*ContextConfig
+
+type ContextConfig struct {
+	Context string        `yaml:"context"`
+	Values  ContextValues `yaml:"values"`
+}
+
+type ContextValues struct {
+	DefaultRepo string `yaml:"default-repo,omitempty"`
```

Comment:
What does this do?

Reply:
Nothing in this change, this would be one of the first values we add to the config though. Removed.